service/src/main/java/org/apache/griffin/core/event/GriffinEventListeners.java

```diff
@@ -0,0 +1,51 @@
+package org.apache.griffin.core.event;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Configuration
+@ConfigurationProperties(prefix = "internal.event")
```

Comment:
I'm not sure you need it here, this will map all class fields (that have setters and getters) to properties.

Reply:
right. @toyboxman 
ConfigurationProperties mainly for configuration bean, I see you have used Value, which make sense since you have only one value properties here.
So I agree with @gavlyukovskiy,  I don't think we need support configuration bean here.

Reply:
thanks guys

I'm gonna remove @ ConfigurationProperties annotation